### PR TITLE
Update Travis CI to use standard test script.

### PR DESCRIPTION
**Pull request by kevina**
Also remove old go versions from .travis.yml.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 language: go
 
 go:
-  - 1.3
-  - 1.4
   - release
```
Comment (Stebalien) [inline .travis.yml]: I'd explicitly set 1.9.x (yes, we'll need to update it but it forces us to re-run CI when doing so).

**Review by Stebalien — CHANGES_REQUESTED** [review]

**Review by Stebalien — APPROVED** [review]

**Comment by kevina** [comment]
Note, Travis CI is not enabled for this repo...

**Comment by kevina** [comment]
@Stebalien let's not merge this one just yet.  I think it might need a rebase.  I will look into it later today.

**Comment by kevina** [comment]
@Stebalien okay we are green now

**Pull request merged**